test/functional/interface_zmq.py

```diff
@@ -13,10 +13,9 @@
 from test_framework.util import (
     assert_equal,
     hash256,
+    connect_nodes,
```

Comment:
16404: sort

Reply:
Have Updated this file , not sure why this is not marked as "Outdated"